cmd/tsbs_generate_data/main.go

```diff
@@ -208,6 +223,9 @@ func main() {
 	if ok := validateFormat(format); !ok {
 		fatal("invalid format specifier: %v (valid choices: %v)", format, formatChoices)
 	}
+	if ok := validateUseCase(useCase); !ok {
+		fatal("invalid format specifier: %v (valid choices: %v)", useCase, useCaseChoices)
```

Comment:
should be `invalid useCase specified:`

(also just realized the above message has a typo too, with `specifier` instead of `specified`, can you fix too? :) )